### PR TITLE
added test on a deletion of a newly created project

### PR DESCRIPTION
**Pull request by vklonin**
added the test: "checking project deletion process", 
menu items enum extended with item "SETTINGS", 
ProjectTable added a public variable with list size.

Test-case description:

> **Preconditions**

- user logged in to the allure.testOps
- new project with **known** name is created

> **Actions**

- From the project page navigates in Sidebar to Settings
- Push delete button and confirm a deletion

> **Assert**

- Search for the project by **known** name
- Ascertain that the project list after this search is empty

### CODE REVIEW

File: src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import static org.assertj.core.api.Assertions.*;
+
+@Story("Projects tests dynamic interactions")
+public class ProjectTestsDynamic extends TestBase{
+
+    @WithLogin
+    @Test
+    void deleteProjectCreatedInThisTest(){
+        //precondition
+        String projectName = "testuser-testproject-toBeDeleted";
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java:22]: добавить рандомное имя, java faker / RandomUtils

File: src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        String projectAbbr = "TD";
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java:23]: naming

File: src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+        step("Open main page", () ->
+             open(""));
+        step("click on a button 'New project'", () ->
+             $$("button.Button_style_success").find(Condition.text("New project")).click());
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java:28]: упростить

File: src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+             $("input[name=name]").setValue(projectName);
+             $("input[name=abbr]").setValue(projectAbbr);
+        });
+        step("click submit, creating {projectName} project", () ->
+            $("button.Button_style_success[type=submit]").click()
+        );
+        //test
+        step("navigate to 'settings' in sidebar and click", () -> {
+            Sidebar sidebar = new Sidebar();
+            sidebar.navigateTo(MenuItem.SETTINGS);
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/tests/ProjectTestsDynamic.java:39]: это не PO феншуй

File: src/test/java/cloud/autotests/pages/ProjectsTable.java [hunk 5]
```diff
@@ -12,10 +12,12 @@
 public class ProjectsTable {
 
     private ElementsCollection tableRows = $$("li.list-row");
+    public int size;
```
Comment (svasenkov) [inline src/test/java/cloud/autotests/pages/ProjectsTable.java:15]: убрать либо в метод checkTableSize(Integer size) {

}

**Review by svasenkov — CHANGES_REQUESTED** [review]

**Comment by vklonin** [comment]
Перепишу все под PO с учетом замечаний

**Pull request closed without merging**